tutorials/scripting/gdextension/gdextension_docs_generation.rst

```diff
@@ -0,0 +1,47 @@
+.. _doc_gdextension_docs_generation:
+
+Adding documentation to your GDExtension
+========================================
+
+.. note::
+    Adding documentation for GDExtensions is only possible for 4.3 and later.
```

Comment:
Is it only displaying it, or generating the files for it that is only possible with Godot 4.3.

In other words, if I want to generate docs for a GDExtension, whose `compatibility_minimum = "4.2"`, can I generate the stub doc files and then view docs with Godot 4.3-rc2?

Reply:
Good point 👍🏻 the generation is also only supported in 4.3+

Reply:
If I add documentation comments in an GDExtension, can users on Godot 4.2 still use the extension if `compatibility_minimum` allows? This should probably be made clear, just in case.

Reply:
@Calinou Yes that should still work. For 4.2 users they just won't have the documentation feature and will get an empty Node "documentation" shown